src/commands/Tools/Websearch/igdb.ts

```diff
@@ -144,4 +143,24 @@ export default class extends RichDisplayCommand {
 		if (!platforms || isArrayOfNumbers(platforms)) return fallback;
 		return platforms.map((platform) => platform.name).join(', ');
 	}
+
+	private buildBody(game: string) {
+		return [
+			[`search "${game}"`, 'limit 10', 'offset 0;'].join('; '),
+			`fields ${[
+				'name',
+				'url',
+				'summary',
+				'rating',
+				'involved_companies.developer',
+				'involved_companies.company.name',
+				'genres.name',
+				'release_dates.date',
+				'platforms.name',
+				'cover.url',
+				'age_ratings.rating',
+				'age_ratings.category;'
+			].join(',')}`
```

Comment:
Please make this a property in the class so we don't have to create an array and join it every time we call this.